fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -3100,10 +3100,16 @@ public static int metaServiceRpcRetryTimes() {
     @ConfField(mutable = true)
     public static int audit_event_log_queue_size = 250000;
 
-    @ConfField(description = {"存算分离模式下streamload导入使用的转发策略, 可选值为public-private或者空",
-            "streamload route policy in cloud mode, availale options are public-private and empty string"})
+    @ConfField(mutable = true, description = {"streamload导入使用的转发策略, 可选值为public-private/public/private/random_be/空",
+            "streamload route policy, availale options are public-private/public/private/random_be and empty string"})
     public static String streamload_redirect_policy = "";
 
+    @ConfField(mutable = true, description = {"be的公网endpoint", "be public endpoint"})
+    public static String be_public_endpoint = "";
+
+    @ConfField(mutable = true, description = {"be的私网endpoint", "be private endpoint"})
+    public static String be_private_endpoint = "";
+
```

Comment:
Seems confusing. We should alter backend properties("tag.be_public_endpoint" = "xxx" ... ) for each backend to set the endpoints instead of sharing the same endpoint for default behaviors

Reply:
In compute-storage integrated scenarios, it's convenient to configure a global VIP to avoid having to modify each time when adding a BE , which could lead to omissions.